docs/tutorials/galga.md

```diff
@@ -58,7 +58,7 @@ spacePlane = sprites.create(img`
 
 ## Step 2
 
-Go back to ``||sprites:Sprites||`` again and pull out a ``||sprites:set mySprite stay in screen||`` and put it in after the other sprite block. Change the ``||variables:mySprite||`` to ``||variables:spacePlane||``. Click the ``OFF`` button to make it switch to ``ON``. Go over to ``||info:Info||``, get a ``||info:set life to||`` block, and put it in there too. Set the life count to `3`.
+Go back to ``||sprites:Sprites||`` again and pull out a ``||sprites:set mySprite stay in screen||`` and put it in after the other sprite block. Change ``||variables:mySprite||`` to ``||variables:spacePlane||``. Click the ``OFF`` button to make it switch to ``ON``. Go over to ``||info:Info||``, get a ``||info:set life to||`` block, and put it in there, too. Set the life count to `3`.
```

Comment:
Emphatic comma use...this is personal preference too...or personal preference, too.

Interesting discussion about this: https://www.quickanddirtytips.com/education/grammar/when-use-comma-too

Reply:
I love reading Grammar Girl's work (although she didn't author this particular article). I typically defer to Chicago. Interesting that Bonnie references Chicago for the article, since it states that _too_ should be set-off with commas.